README.md

```diff
@@ -60,34 +69,53 @@ make -C ${ZKLLVM_BUILD:-build} assigner clang -j$(nproc)
 
 zkLLVM's workflow is as follows:
 
-1. Users wishing to prove some statement are supposed to implement an application in a language compatible with some frontend (C++ for now). This code will be compiled with a modified version of the `clang` compiler, which will output intermediate representation of the circuit. 
-   > For the most performant cryptography circuits (e.g. hashes, signatures, VDFs, proof system verifications, etc.) 
-   > we recommend using [=nil; Foundation's Crypto3 library](https://github.com/nilfoundation/crypto3.git).
+> TODO: are statement and circuit the same thing? 
+> Why do we use both 'prove a statement' and
+> 'generate a proof of the circuit'?
+
+1.  Users wishing to prove a statement should implement an application in a language
+    compatible with some frontend (C++ for now).
+    This code will be compiled with a modified version of the `clang` compiler,
+    which will output an intermediate representation of the circuit. 
 
-2. zkLLVM is tightly coupled with [=nil; Foundation's Proof Market](https://proof.market.nil.foundation) so users willing to generate a 
-   proof for the circuit, will be required to put an order for a proof so proof generators could pick it up and 
-   generate proofs necessary. To submit an order it is required to: 
+    > We recommend using [=nil; Foundation's Crypto3 library](https://github.com/nilfoundation/crypto3.git)
+    > for the most performant cryptography circuits: hashes, signatures, VDFs, proof system verifications, and so on.
 
-    * Retrieve a constraints description from Constraint : Binary file representing arithmetization of the circuit.
-    * Assignment Table: Binary file pre-processed public inputs & witness.
+2.  Users put an order for proof on the [=nil; Foundation's Proof Market](https://proof.market.nil.foundation).
+3.  Proof generators pick up the orders and generate proofs.
 
-   The constraint and assignment table generated above should be passed as in input to proof generator binary. This 
-   will output a binary proof file.
+An order consists of two binary files:
 
-3. Proof verification is not part of the zkLLVM project. This involves a few more steps which requires serialization of
-   the circuit to be setup on chain.
+*   Constraint representing the arithmetization of the circuit.
+*   Assignment Table contains pre-processed public inputs and witness.
+
+The constraint and assignment table are passed as in input to proof generator binary,
+which then outputs a binary proof file.
+
+Proof verification is not a part of the zkLLVM project.
+It involves a few more steps which require circuit serialization 
+to be set up on the blockchain.
 
 ### Examples
 
 #### Linux 
 
 ```bash
 make -C ${ZKLLVM_BUILD:-build} circuit_examples -j$(nproc) 
-${ZKLLVM_BUILD:-build}/bin/assigner/assigner -b ${ZKLLVM_BUILD:-build}/examples/arithmetics_example.bc -i examples/arithmetics.inp -t assignment.tbl -c circuit.crct
+${ZKLLVM_BUILD:-build}/bin/assigner/assigner \
```

Comment:
A few words of explanation, maybe? Like, there's an example in this repository. Now we will compile it, calculate a statement, then prove it, and then check the proof.